flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/window/tvf/slicing/SliceAssigners.java

```diff
@@ -69,6 +69,12 @@ public static TumblingSliceAssigner tumbling(
         return new TumblingSliceAssigner(rowtimeIndex, shiftTimeZone, size.toMillis(), 0);
     }
 
+    public static TumblingSliceAssigner tumbling(
```

Comment:
Although this simplifies the original `tumbling(...).withOffset(...)`, I think we'd better not introduce new methods if they are only used for testing purposes, as it would slightly increase maintenance, WDYT?